test/unit/single_address_test.rb

```diff
@@ -28,10 +28,20 @@ class SingleAddressTest < ActiveSupport::TestCase
       assert_equal("", single)
     end
 
+    should "dump empty values as nil" do
+      single = EmailAttribute::SingleAddress.new(nil)
+      assert_nil(EmailAttribute::SingleAddress.dump(single))
+    end
+
+    should "load nil values as empty addresses" do
+      single = EmailAttribute::SingleAddress.load(nil)
+      assert_equal("", single)
```

Comment:
I don't think so -- if you look at the tests above, the current behavior is for `EmailAttribute::SingleAddress` to always return some form of string.